packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,571 @@
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : Record AppMaps ->
+          </button>
+        </div>
+      </section>
+      <section class="qs-step" v-if="currentStep === 3">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Record AppMaps</h1>
+          <select class="qs-select">
+            <option>minitest (detected)</option>
+          </select>
+        </div>
+        <div class="qs-step__block" v-if="!step3Completed">
+          <p>
+            An easy way to create AppMaps is by running your tests. This will
+            run a standard command to run your tests and generate AppMap data.
+          </p>
+          <code class="qs-code" @click="select"
+            >APPMAP=true bundle exec rake test</code
+          >
+          <button class="qs-button" v-if="!isActionRunning" @click="runAction">
+            Run tests to create AppMaps
+          </button>
+          <div
+            class="qs-loader"
+            v-if="isActionRunning"
+            data-process="5 AppMaps created"
+          >
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__success" v-if="step3Completed">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            AppMaps recorded
+          </span>
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : View AppMaps ->
+          </button>
+        </div>
+      </section>
+    </div>
+    <div class="qs-help">
+      <HelpIcon class="qs-help__icon" />
+      <div class="qs-help__text">
+        <p>Need help or want to give feedback?</p>
+        <a
+          href="https://appland.com/company/contact-us"
+          target="_blank"
+          rel="noopener noreferrer"
+          >Contact an AppLand developer directly.</a
+        >
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import HelpIcon from '@/assets/quickstart/help.svg';
+import StepLeftIcon from '@/assets/quickstart/arrow-left.svg';
+import StepRightIcon from '@/assets/quickstart/arrow-right.svg';
+import SuccessIcon from '@/assets/quickstart/success.svg';
+
+export const Steps = {
+  INSTALL_AGENT: 1,
+  CONFIGURE_APPMAP: 2,
+  RECORD_APPMAPS: 3,
+};
+
+export default {
+  name: 'Quickstart',
+
+  components: {
+    HelpIcon,
+    StepLeftIcon,
+    StepRightIcon,
+    SuccessIcon,
+  },
+
+  props: {
+    language: {
+      type: String,
+      default: null,
+    },
+    initialStep: {
+      type: Number,
+      default: 1,
+    },
+    completedSteps: {
+      type: Array,
+      default: () => [],
+    },
```

Comment:
Can we update the content of the `completedSteps` array? The milestones have three states: `complete` | `incomplete` | `error`.

Instead of an array of completed steps, the array should hold the state of all steps.
```
[  'complete', 'complete', 'error', 'incomplete' ]
```